packages/core/src/components/way-input/way-input.scss

```diff
@@ -0,0 +1,141 @@
+   *
+   * @prop --placeholder-color: Text color of the placeholder
+   *
+   * @prop --min-height: Minimum height of the select
+   *
+   * @prop --label-margin-start: Left margin of the label (will be right margin when we support right-to-left direction)
+   * @prop --label-margin-end: Right margin of the label (will be left margin when we support right-to-left direction)
+   *
+   * @prop --clear-icon-margin-end: Right margin of the clear icon (will be left margin when we support right-to-left direction)
+   *
+   * @prop --caret-margin-end: Right margin of the caret (will be left margin when we support right-to-left direction)
+   *
+   * @prop --tags-padding-top: Top padding of the tags
+   * @prop --tags-padding-bottom: Bottom padding of the tags
+   *
+   * @prop --tags-margin-end: Right margin of the tags (will be left margin when we support right-to-left direction)
+   */
+  --font-size: var(--way-form-element-font-size-medium);
+  --font-weight: var(--way-font-weight-normal);
+
+  --background-color: var(--way-color-white);
+  --background-color-hover: var(--way-color-white);
+
+  --border-radius: var(--way-form-element-border-radius-small);
+  --border-color: var(--way-color-light);
+  --border-color-hover: var(--way-color-medium);
+
+  --color: var(--way-color-dark-tint);
+
+  --placeholder-color: var(--way-color-medium-tint);
+
+  --min-height: var(--way-form-element-height-medium);
+
+  --label-margin-start: var(--way-spacing-medium);
+  --label-margin-end: var(--way-spacing-medium);
+
+  --clear-icon-margin-end: var(--way-spacing-medium);
+
+  --caret-margin-end: var(--way-spacing-medium);
+
+  --tags-padding-top: 3px;
+  --tags-padding-bottom: 3px;
+
+  --tags-margin-end: var(--way-spacing-xx-small);
+
+  display: block;
+
+  box-sizing: border-box;
+
+  & *,
+  & *:before,
+  & *:after {
+    box-sizing: inherit;
+  }
+}
+
+// Size modifiers
+// --------------------------------------------------
+.input-small {
+  --font-size: var(--way-form-element-font-size-small);
+  --min-height: var(--way-form-element-height-small);
+
+  --label-margin-start: var(--way-spacing-small);
+  --label-margin-end: var(--way-spacing-small);
+}
+
+.input-large {
+  --font-size: var(--way-form-element-font-size-large);
+  --min-height: var(--way-form-element-height-large);
+
+  --label-margin-start: var(--way-spacing-large);
+  --label-margin-end: var(--way-spacing-large);
+}
+
+.input {
+  display: block;
+  font-family: var(--way-font-family);
+  font-size: var(--font-size);
+  font-weight: var(--font-weight);
+  letter-spacing: normal;
+  background-color: var(--background-color);
+  width: 100%;
+  border: solid 1px var(--border-color);
+  border-radius: var(--border-radius);
+  min-height: var(--min-height);
+  color: var(--color);
+  overflow: hidden;
+  transition: 150ms color, 150ms border, 150ms box-shadow;
+  padding: var(--way-spacing-x-small) var(--way-spacing-small);
+}
+
+.input:not(.input-disabled) .input:hover {
```

Comment:
**issue**: the hover doesn't work because this expects an .input underneath another .input. I will fix this.